jena-tdb/src/test/java/org/apache/jena/tdb/base/file/TestLocationLock.java

```diff
@@ -146,4 +146,34 @@ public void location_lock_dir_error_02() throws IOException {
         Assert.assertFalse(lock.canObtain());
         lock.release();
     }
+
+    @Test
+    public void location_lock_dir_error_03() throws IOException {
+        Assume.assumeTrue(negativePidsTreatedAsAlive);
+
+        Location dir = Location.create(tempDir.getRoot().getAbsolutePath());
+        LocationLock lock = dir.getLock();
+        Assert.assertTrue(lock.canLock());
+        Assert.assertFalse(lock.isLocked());
+        Assert.assertFalse(lock.isOwned());
+        Assert.assertTrue(lock.canObtain());
+
+        // Write a TDB2 format lock file
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(dir.getPath("tdb.lock")))) {
+            // TDB2 format lock file, this writes a new line to the end of the lock file
+            writer.write(Integer.toString(-1234));
+            writer.write('\n');
+        }
+
+        // Trying to get the owner should error accordingly
+        try {
+            lock.canObtain();
+            Assert.fail("Expected the lock file to be considered invalid");
+        } catch (FileException e) {
+            String errMsg = e.getMessage();
+            Assert.assertNotNull(errMsg);
+            System.out.println(errMsg);
```

Comment:
Is this `System.out.println` leftover from debugging?

Reply:
Guilty!